templates/email/default/waste/csc_payment_failed.html

```diff
@@ -0,0 +1,25 @@
+[%
+
+email_summary = "There was a problem with your payment";
+email_columns = 2;
+
+PROCESS '_email_settings.html';
+INCLUDE '_email_top.html';
+
+%]
+
+<th style="[% td_style %][% primary_column_style %]" id="primary_column">
+  [% start_padded_box | safe %]
+  <h1 style="[% h1_style %]">There was a problem with your payment</h1>
+  <p style="[% p_style %]">There was a problem taking payment for your [% report.title %]</p>
+
+  <p style="[% p_style %]">Please contact the Customer Service Centre to try again.</p>
```

Comment:
Might be nice to inform the user how they can do this, especially as it's sent from a no-reply email address.

Reply:
I need to get Bromley to confirm all the text in the emails so this is mostly a place holder.